sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCV2Suite.scala

```diff
@@ -2692,4 +2692,19 @@ class JDBCV2Suite extends QueryTest with SharedSparkSession with ExplainSuiteHel
       JdbcDialects.registerDialect(H2Dialect)
     }
   }
+
+  test("SPARK-41516: Allow jdbc dialects to override the query used to create a table") {
```

Comment:
where does the override happen in this test?

Reply:
updated